client_test.go

```diff
@@ -101,6 +101,7 @@ func TestCopyToLocal(t *testing.T) {
 func TestCopyToRemote(t *testing.T) {
 	client := getClient(t)
 
+	baleet(t, "/_test/copytoremote.txt")
```

Comment:
This was added to prevent errors on multiple test runs against the same hdfs cluster.